src/frontend/src/pages/AdminToolsPage/TeamsTools.tsx

```diff
@@ -89,6 +95,57 @@ const TeamsTools = () => {
     </TableRow>
   ));
 
+  interface DescriptionButtonsProps {
+    toRight: ReactNode;
+  }
+
+  const DescriptionButtons: React.FC<DescriptionButtonsProps> = ({ toRight }) => {
```

Comment:
Calling this DescriptionButtons isn't quite right since it's really there to hold the header, go ahead and rename it